Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -9,11 +9,16 @@ extension Environment {
     database: .mock,
     envVars: EnvVars(),
     gitHub: .mock,
+    logger: .mock,
     sendEmail: const(pure(.init(id: "deadbeef", message: "success!"))),
     stripe: .mock
   )
 }
 
+extension Logger {
+  public static let mock = Logger(level: .debug, logger: { _ in })
```

Comment:
this ok to do for tests? so we dont get flooded with logs

Reply:
Yep looks good!

Reply:
We could even make a buffer and write tests/snapshots against our logging!

Reply:
yeah that'd be super cool